### ISSUE TITLE
Wrong orders in colorname

### ISSUE THREAD

**Issue by fikovnik** [honza/base16-st#2]
In st 0.8.1, it seems that that the order in the `colorname` is wrong or at least it is not compatible with base16-shell or with base16 emacs themes. It makes the themes to pick wrong color as background. Easy way to test is to use any dark theme and then run base16-shell with the same theme.

The correct should be:
```c
static const char *colorname[] = {
  "#1b2b34", /* base00 */
  "#ec5f67", /* base08 */
  "#99c794", /* base0B */
  "#fac863", /* base0A */
  "#6699cc", /* base0D */
  "#c594c5", /* base0E */
  "#5fb3b3", /* base0C */
  "#c0c5ce", /* base05 */

  "#65737e", /* base03 */
  "#f99157", /* base09 */
  "#343d46", /* base01 */
  "#4f5b66", /* base02 */
  "#a7adba", /* base04 */
  "#cdd3de", /* base06 */
  "#ab7967", /* base0F */
  "#d8dee9", /* base07 */
};

unsigned int defaultfg = 7;
unsigned int defaultbg = 0;
```

**Comment by honza** [comment]
Interesting! It seems that I changed the values of `defaultfg` and `defaultbg` in my config file, and didn't even realize. I synced my config file with upstream, and applied the color values in the order that you propose. It's indeed better but there still seems to be an issue with a dark black cursor. I'm going to investigate a bit more before making further changes.

Thanks!

**Comment by honza** [comment]
This is very interesting. It works perfectly for solarized, but it's broken for monokai, and I suspect others, too.

**Comment by honza** [comment]
Fixed in eff195986fa8be8b030943a16e7f9ab95653f2a9

Give a try!
